cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -385,6 +390,16 @@ public RunId run(ProgramId programId, Map<String, String> overrides, boolean deb
     if (overrides != null) {
       userArgs.putAll(overrides);
     }
+
+    if ((userArgs.containsKey(RUNTIME_ARG_KEYTAB)) &&
+            (userArgs.containsKey(RUNTIME_ARG_PRINCIPAL))) {
+        String principal = userArgs.get(RUNTIME_ARG_PRINCIPAL);
+        LOG.debug("Checking authorisation for user: " + authenticationContext.getPrincipal() +
```

Comment:
Use place holder `{}` instead of string concatenation for logging.

Reply:
Fixed